### PR TITLE
Shortens and rephrase

### PR DESCRIPTION
**Pull request by FlorianJacta**
No real content of the Getting Started has been changed. It is just shorter and more user-friendly.

### CODE REVIEW

File: docs/getting_started/index.md [hunk 1]
```diff
@@ -5,46 +5,35 @@ hide:
...
-Welcome to Taipy! This guide will show you how to install Taipy and create your first application.
+Dive into Taipy with this beginner-friendly guide. Learn to install, configure, and deploy your first application with ease.
 
-## Install Taipy with pip
-
-To begin with Taipy, ensure you have Python installed (version 3.8 or higher) along with
-[pip](https://pip.pypa.io). Once you have these prerequisites, open your terminal or command 
-prompt and run the following command to get the latest stable version of Taipy downloaded and 
-installed:
+## Installation
```
Comment (jrobinAV) [inline docs/getting_started/index.md]: I prefer "Install Taipy with pip", or "Installation with pip"

File: docs/getting_started/index.md [hunk 2]
```diff
@@ -5,46 +5,35 @@ hide:
...
-    if you require different ways to install Taipy or don't have Python or pip installed.
 
+    For alternative installation methods or if you're lacking Python or pip, refer to the [installation page](../installation/index.md).
 
-## First Taipy Scenario 
-Taipy introduces the concept of a *Scenario* to model pipeline executions. A *Scenario* consists 
-of an execution graph, often represented as a Directed Acyclic Graph (DAG), where tasks (or 
-functions) exchange data. You can make your scenario as easy or as complicated as you want, 
-depending on what you need.
+## Dive into Your First Taipy Scenario
```
Comment (jrobinAV) [inline docs/getting_started/index.md]: You already used the metaphor "Dive into" in the very first line. I believe it s repetitive.

File: docs/getting_started/index.md [hunk 3]
```diff
@@ -61,35 +50,23 @@ scenario_cfg = Config.configure_scenario("scenario", task_configs=[build_msg_tas
...
 - Finally, line 11 configures the execution graph of the scenario providing 
   the previously configured task.
 
-### Core service run
-
-The Core service processes the configuration made in the previous step to set up the scenario 
-management feature.
-
-Include the following code in your Python script:
+2. **Core Service Initialization**: The Core service processes the configuration of the previous step to set up the scenario management feature.
```
Comment (jrobinAV) [inline docs/getting_started/index.md:59]: Well, it's not an initialization but a run.

File: docs/getting_started/index.md [hunk 4]
```diff
@@ -61,35 +50,23 @@ scenario_cfg = Config.configure_scenario("scenario", task_configs=[build_msg_tas
...
 from taipy import Core
 
 if __name__ == "__main__":
     Core().run()
 ```
-- Line 3 is standard boilerplate code to ensure your script runs only from the main module, 
-preventing accidental execution. It is strongly recommended to include it.
 
-- Line 4 instantiates and runs a Core service.
+3. **Scenario Creation & Data Management**: With the Taipy Core service up and running, you can create and manage scenarios, submit task graphs for execution, and access data nodes:
```
Comment (jrobinAV) [inline docs/getting_started/index.md]:
```suggestion
3. **Scenario & Data Management**: With the Core service up and running, you can create
and manage scenarios, submit task graphs for execution, and access data nodes:
```

File: docs/getting_started/index.md [hunk 5]
```diff
@@ -112,39 +89,33 @@ and writes the result to the output data node.
...
 
-And here is the expected output:
+Expected Output:
 ``` console
 [2023-02-08 20:19:35,062][Taipy][INFO] job JOB_build_msg_9e5a5c28-6c3e-4b59-831d-fcc8b43f882e is completed.
 Hello Taipy!
 ```
 
-## Graphical Interface
+## A Glimpse into creating a GUI
```
Comment (jrobinAV) [inline docs/getting_started/index.md]:
```suggestion
## Build a graphical interface
```
or building a graphical interface.
I believe it is better to have consistent formulations in the titles. Only names, or only verbs.

File: docs/getting_started/index.md [hunk 6]
```diff
@@ -112,39 +89,33 @@ and writes the result to the output data node.
...
 
-In the previous step, you used Taipy Python APIs to access, edit, and submit a scenario.
-This interaction is typically integrated into Python functions that get triggered by end-user 
-actions in a graphical interface built using Taipy. 
+While we used the Python APIs of Taipy to manage our scenario, it typically also integrates with a graphical interface, also built using Taipy, for smoother user interactions. Here's a basic GUI setup for our "Hello World" scenario:
```
Comment (jrobinAV) [inline docs/getting_started/index.md]:
```suggestion
While we used the Python APIs of Taipy to manage our scenario, it typically integrates
with a graphical interface, also built using Taipy, for end-user interactions. Here's a
basic GUI set up for our "Hello World" scenario:
```

File: docs/getting_started/index.md [hunk 7]
```diff
@@ -156,15 +127,14 @@ if __name__ == "__main__":
     Gui(page).run()
 ```
 
-TODO: explain the code above and result below
+For the full example with GUI, download:
```
Comment (jrobinAV) [inline docs/getting_started/index.md]:
```suggestion
For the full example with the user interface code, download:
```

File: docs/getting_started/index.md [hunk 8]
```diff
@@ -156,15 +127,14 @@ if __name__ == "__main__":
...
+![GUI Result](result.png){width=50%}
+
+
+---
 
-That covers the fundamental steps to begin with Taipy. For more realistic 
-use cases and advanced features, refer to the 
-[tutorials](../tutorials/index.md), [demos](../demos/index.md), or 
-[manuals](../manuals/index.md) pages.
+For deeper dives and more intricate use cases, check out our [Tutorials](../tutorials/index.md), [Demos](../demos/index.md), or [Manuals](../manuals/index.md).
```
Comment (jrobinAV) [inline docs/getting_started/index.md]: Yet another time using the dive metaphor...
```suggestion
For more realistic and advanced use cases, check out our [Tutorials](../tutorials/index.md), [Demos](../demos/index.md), or [Manuals](../manuals/index.md).
```

File: docs/getting_started/index.md (hunk 3 above)
Comment (FlorianJacta) [reply docs/getting_started/index.md:59]: Even if it is not technical, don't you think it has a good english meaning and explains well what's happening?

Comment (jrobinAV) [reply docs/getting_started/index.md:59]: I prefer 'run' for 2 reasons: 
1. Even if we don't make it explicit, 'run' suggests that there is something 'alive'. A process, a service, or whatever. The term 'initialization' implies that once it is initialized, nothing happens, which is incorrect.
2. It is consistent with the method name.

**Review by jrobinAV — APPROVED** [review]

File: docs/getting_started/index.md (hunk 3 above)
Comment (FlorianJacta) [reply docs/getting_started/index.md:59]: I don't really like it, to be honest. 'Core Service Run' needs some meaning to me.

For 2 reasons:

This is technically what is happening, but as the user sees it, it initializes the Core with the configuration loaded, and now he can create scenarios and tasks, ...

Run also has some other meaning in Data Science for pipeline runs for example.

I won't fight for it.

Comment (jrobinAV) [reply docs/getting_started/index.md:59]: I won't either...
I approved the PR anyway. It's no blocking to me.

File: docs/getting_started/index.md [hunk 9]
```diff
@@ -5,46 +5,35 @@ hide:
...
-Taipy introduces the concept of a *Scenario* to model pipeline executions. A *Scenario* consists 
-of an execution graph, often represented as a Directed Acyclic Graph (DAG), where tasks (or 
-functions) exchange data. You can make your scenario as easy or as complicated as you want, 
-depending on what you need.
+## Your First Taipy Scenario
 
-For your first scenario, we'll create a simple "Hello World" example to show you the 
-configuration process. Here's a picture of the scenario's execution graph, which has two data 
-nodes (blue boxes) and one task (orange box):
+A Taipy *Scenario* models pipeline executions. Think of it as an execution graph, typically a Directed Acyclic Graph (DAG), with tasks or functions sharing data. The complexity of your scenario is all up to you.
```
Comment (erikfTW) [inline docs/getting_started/index.md]:
```suggestion
In Taipy, a *Scenario* represents how pipeline executions work. Imagine it as an execution graph, often like a Directed Acyclic Graph (DAG), where tasks or functions work together and share data. How complex your scenario becomes is entirely in your hands.
```

File: docs/getting_started/index.md [hunk 10]
```diff
@@ -112,39 +89,33 @@ and writes the result to the output data node.
 - Line 6 reads and prints the output data node *message* written by the execution of the scenario 
 *hello_scenario*.
 
-Putting all together the pieces, you can download the entire Python code for the example
-from this link: <a href="./hello_world_scenario.py" download>`hello_world_scenario.py`</a>
+For convenience, grab the entire code for this example:
```
Comment (erikfTW) [inline docs/getting_started/index.md]:
```suggestion
For your convenience, you can get the complete code for this example:
```

File: docs/getting_started/index.md (hunk 6 above)
Comment (erikfTW) [inline docs/getting_started/index.md]:
```suggestion
While we've used Taipy's Python APIs to handle our scenario, it usually works seamlessly with a graphical interface, also created using Taipy, to provide a more user-friendly experience. Here's a simple GUI set up for our "Hello World" scenario:
```

File: docs/getting_started/index.md (hunk 7 above)
Comment (erikfTW) [inline docs/getting_started/index.md]:
```suggestion
To get the complete example, including the user interface code, download it here:
```

**Review by erikfTW — CHANGES_REQUESTED** [review]

File: docs/getting_started/index.md (hunk 9 above)
Comment (FlorianJacta) [reply docs/getting_started/index.md]: I am not sure of the way it is phrased.

Comment (erikfTW) [reply docs/getting_started/index.md]: Then I would try: 

A Taipy *Scenario* models pipeline executions. Think of it as an execution graph where tasks or functions collaborate and exchange data. You have full control over how complex your scenario can be.

**Review by erikfTW — APPROVED** [review]

**Pull request merged**